src/domain/behaviours/BacktrackingBehaviour/BasicBacktrackingBehaviour.ts

```diff
@@ -122,8 +136,8 @@ export class BasicBacktrackingBehaviour implements IBackTrackingBehaviour {
     return intx
   }
 
-  async peek(steps = 0, context: IContext = this.context, fromLeft = false): Promise<IRichCursorInputRequired> {
-    const intx = this._findInteractiveInteractionAt(steps, context, fromLeft)
+  async peek(steps = 0, context: IContext = this.context, direction = PeekDirection.FROM_RIGHT): Promise<IRichCursorInputRequired> {
```

Comment:
Let's say we are calling `peek(1, context, PeekDirection.FROM_RIGHT)`. If we were to say this in english, we are saying is `Peek one position to the left, as I am on the right side`. It kinda reads like a "comeFrom" statement instead of a "goto" statement, which is absolutely the same thing, but the logic of "comeFrom" is so much harder to wrap our heads around. Should we actually be saying just `PeekDirection.RIGHT` or `PeekDirection.LEFT`? Or maybe I am misunderstanding something?

Reply:
Yeah, especially since the enum denotes the subject of `Direction` 👌